core/src/main/java/io/ddf/analytics/AggregationHandler.java

```diff
@@ -122,31 +103,37 @@ public DDF groupBy(List<String> groupedColumns) {
     return this.getDDF();
   }
 
+  protected String buildGroupBySQL(List<String> aggregateFunctions) throws DDFException {
+    String groupedColSql = Joiner.on(",").join(mGroupedColumns);
+
+    List<String> aggregationFuncSql = new ArrayList<String>();
+    for (int i = 0; i < aggregateFunctions.size(); i++) {
+      aggregationFuncSql.add(convertAggregateFunctionsToSql(aggregateFunctions.get(i)));
+    }
+
+    String selectFuncSql = Joiner.on(",").join(aggregateFunctions);
```

Comment:
@lebinh That Google Function does not worth to use here in Java code. I refactor a bit for easier understanding. Comment? :)